lib/flux/app-state.js

```diff
@@ -136,7 +136,9 @@ export const actionMap = new ActionMap({
     newTag: {
       creator({ tagBucket, name }) {
         return () => {
-          tagBucket.add({ name }, () => {
+          // tag.id must be the tag name in lower case and percent escaped
+          const tagId = encodeURIComponent(name.toLowerCase());
+          tagBucket.update(tagId, { name }, () => {
             // dispatch( this.action( 'loadTags', { tagBucket } ) );
```

Comment:
what's the point of this empty function? should we just go ahead and remove it now? should we reload the tags here?

Reply:
I removed it in c6b0f8e. The tags list appears to update fine after adding the tag so I don't think it's needed.